### ISSUE TITLE
Bad metadata on parsec 3.6

### ISSUE THREAD

**Issue by rohan** [sighingnow/parsec.py#33]
When installing parsec in a fresh python3 virtualenv, I get the following error:
```
❯ pip install --no-cache-dir parsec
Collecting parsec
  Using cached parsec-3.6-py2.py3-none-any.whl
ERROR: Requested parsec from https://files.pythonhosted.org/packages/fa/d2/2a80bd6b13e25d4fdd04b15bb182acc61a0f4cfd60dbb995ab729d351b4f/parsec-3.6-1.tar.gz#sha256=e2f8b412bcd46e3fae8029b58c743cc606a411b1e1770965c8da08d02c754b22 has different version in metadata: '3.6'
```

This causes pip 20.3.3 to hard-fail on installation. Pip 20.3.4 and above just skip 3.6 and downgrade to 3.5, which is also undesirable:

```
❯ pip install --no-cache-dir parsec
Collecting parsec
  Using cached parsec-3.6-py2.py3-none-any.whl
WARNING: Discarding https://files.pythonhosted.org/packages/fa/d2/2a80bd6b13e25d4fdd04b15bb182acc61a0f4cfd60dbb995ab729d351b4f/parsec-3.6-1.tar.gz#sha256=e2f8b412bcd46e3fae8029b58c743cc606a411b1e1770965c8da08d02c754b22 (from https://pypi.org/simple/parsec/). Requested parsec from https://files.pythonhosted.org/packages/fa/d2/2a80bd6b13e25d4fdd04b15bb182acc61a0f4cfd60dbb995ab729d351b4f/parsec-3.6-1.tar.gz#sha256=e2f8b412bcd46e3fae8029b58c743cc606a411b1e1770965c8da08d02c754b22 has different version in metadata: '3.6'  Using cached parsec-3.5-py2.py3-none-any.whl (7.4 kB)
Installing collected packages: parsec
Successfully installed parsec-3.5
```

(Edited comment to use `--no-cache-dir`, just in case.)

**Comment by rohan** [comment]
For others who run into this: you can also avoid the hard-fail on 20.3.3 by passing `--use-deprecated=legacy-resolver`, as described [here](https://github.com/pypa/pip/issues/9203). You can also downgrade to (at least) pip 20.1, or upgrade to 20.3.4.

**Comment by sighingnow** [comment]
Thank you @rohan to let me know that.

That is strange, I think I need some time to figure out the detail, but I promise to that as soon as possible.

**Comment by sighingnow** [comment]
Hi @rohan,

That has been fixed by cut a new release (v3.7) on pypi. Thanks for reporting that.
